insights/parsers/systemd/unitfiles.py

```diff
@@ -45,6 +48,19 @@ def is_on(self, service_name):
             service_name (str): service name including '.service'
 
         Returns:
-            bool: True if service is enabled, False otherwise
+            Union[bool, None]: True if service is enabled, False if it is disabled. None if the
+                               service doesn't exist.
         """
-        return self.services.get(service_name, False)
+        return self.services.get(service_name, None)
```

Comment:
The problem with the former solution was that neither the docstring nor the returned data made any indication of the behavior when the service doesn't exist at all, which turned out to be a slight complication in one of my rules, so I fixed it in 2.x for posterity and left it at its original behavior in 1.x for maximum compatibility.